code/game/turfs/open.dm

```diff
@@ -119,8 +142,10 @@
 					step(C, olddir)
 					C.spin(1,1)
 		else if(lube&SLIDE_ICE)
+			C.slipping = TRUE
 			spawn(1)
 				step(C, olddir)
```

Comment:
@optimumtact Current slipping code does not do the ice slide effect. This is a unique effect for the ice slip. Please start using your actual fucking brains before commenting because i know you have them